src/oscar/templates/oscar/catalogue/browse.html

```diff
@@ -96,7 +96,7 @@ <h4>{% trans "Refine by" %}</h4>
             <div>
                 <ol class="row">
                     {% for product in products %}
-                        <li class="col-xs-6 col-sm-4 col-md-3 col-lg-3">{% render_product product %}</li>
+                        <li class="col-xs-12 col-sm-6 col-md-4 col-lg-3">{% render_product product %}</li>
```

Comment:
`col-xs-*` is not a valid bootstrap class - we can drop `col-xs-12` entirely.

Reply:
Is there a reason for changing the column widths of these elements?

Reply:
`col-xs-*` removed.

Reply:
The reason for changing the column widths - the default font size changed from 14px in Bootstrap 3 to 16px in Bootstrap 4. I kept the coefficients for `h1`, `h2` ..., but now all text bigger in general. With updated column width we will not have text overlapping.